source/docs/software/networktables/creating-sendable-classes.rst

```diff
@@ -121,7 +121,24 @@ Creating an entirely new :code:`Sendable` class follows a similar pattern to the
 
   .. code-tab:: c++
 
-    Todo: add C++ code
+        public class ArmState : public Sendable {
+          private:
+            ArmSubsystem armSubsystem;
+          public:
+            ArmState(ArmSubsystem a) {
+              armSubsystem = a;
+            }
+
+            void initSendable(SendableBuilder s) {
+              /*
+              builder.setSmartDashboardType("ArmState");
+              builder.addDoubleProperty("Angle", () -> armSubsystem::getAngle, (value) -> armSubsystem:setAngle(value));
+              builder.addBooleanProperty("IsOpen", () -> armSubsystem::isOpen, (value) -> armSubsystem:setClaw(value));
+              */
+              Sendable::initSendable(s);
+            }
+        }
```

Comment:
@Oblarg Can you help me fix this example so it makes more sense (and works)? I'm not well versed in object-oriented c++.